RELEASE.md

```diff
@@ -44,3 +44,19 @@ Upgrade to new version:
 ```
 IMG_PREFIX=docker.io/your_username OPERATOR_VERSION=x.y.z OPERATOR_NAMESPACE=openshift-operators make olm-upgrade
 ```
+
+## Test Upgrade using Manifests
```

Comment:
Is this mandatory part of the release process?

Reply:
No, but I wanted to write it down somewhere.
Both manual upgrade steps (OLM and manifests) could be moved to a different file, CONTRIBUTING.md maybe?

Reply:
I've moved it to CONTRIBUTING.md.